qkay/qkay.py

```diff
@@ -18,46 +18,64 @@
 #
 #     https://www.nipreps.org/community/licensing/
 #
+import base64
+import glob
+import json
+import os
+import random
+
+import numpy as np
+import os.path as op
```

Comment:
I think we should transition to Pathlib instead of os and glob, but it's better to address this in a future PR.

Reply:
Good point!